build_prod.py

```diff
@@ -210,9 +253,32 @@ def add_meta_layers(cfg):
             yocto_add_bblayer(cfg, bblayer)
 
 
-def build_init(uri, branch, xml_base_name):
+def build_init(uri, branch, xml_base_name, proposed):
+    if not branch:
+        branch = 'master'
+
     repo_init(uri, branch, xml_base_name)
     repo_sync()
+    c_dir = os.getcwd()
+    # go into prod layer directory to get the repo instance 
+    # and create the remote.
+    # (It is required to apply the pull request).
+    os.chdir('meta-xt-{}'.format(xml_base_name))
+    # Get url of prod-layer to find the pull requests
+    # The name of repository is known but organization - not
+    # The organization can be extracted from theurl of layer 'meta-xt-products'
```

Comment:
s/theurl/the url

Reply:
We hardcode github, products.git etc... So, may we also have "xen-troops" hardcoded as well?
Pull requests are mostly for development which happens on xen-troops.
Or I am missing the point and a developer can do this using his own github account to do the same?

Reply:
I think, we should have a chance to work from own forks and have own meta-xt-products. Now (at least i assume) everybody can setup own environment

Reply:
fixed